### ISSUE TITLE
Annotations are mixed with modifier

### ISSUE THREAD

**Issue by nimakarimipour** [ucr-riple/NullAwayAnnotator#78]
**Describe the bug**
Added annotations are getting mixed with the modifier.

```java
-  @JsonProperty public List<String> bar;
+  @JsonPropertypublic @Nullable  List<String> bar;
```

Or

```
-  @JsonProperty
-  @JsonInclude(JsonInclude.Include.NON_NULL)
-  public String foo;
+  @JsonProperty @JsonInclude(JsonInclude.Include.NON_NULL) @Nullable
+@Nullablepublic   String foo;
```

**To Reproduce**
- The root cause is not detected yet.

**Expected behavior**
```
-  @JsonProperty
-  @JsonInclude(JsonInclude.Include.NON_NULL)
-  public String foo;
+  @JsonProperty @JsonInclude(JsonInclude.Include.NON_NULL) @Nullable
+  public   String foo;
```

and 

```java
-  @JsonProperty public List<String> bar;
+  @JsonProperty public @Nullable  List<String> bar;
```

**Stack trace**
Not-Applicable.

**OS (please complete the following information):**
 - Annotator Version [e.g. v1.3.3-SNAPSHOT]

**Additional context**
This is a subproblem of #77
